db/seeds.rb

```diff
@@ -549,6 +549,51 @@ def require_updates?(real_attrs, seed_attrs)
   is_project: true,
   url: "/web_development_101/project_rails.md"
 )
+# +++++++++++
+# SECTION
+# +++++++++++
+
+section_position += 1
+section = create_or_update_section(
+  title: "Pair Programming",
```

Comment:
I think we should maybe label this section as optional just incase finding a pairing partner is harder than we have anticipated. We will get plenty of feedback on these lessons I'm sure so if we can remove the label if people are embracing pair programming. What do you think @ChadKreutzer?